apps/builder/app/canvas/shared/use-pointer-outline.ts

```diff
@@ -18,6 +18,7 @@ export const usePointerOutline = () => {
     ref.current = div;
     return () => {
       document.body.removeChild(div);
+      ref.current = undefined;
```

Comment:
if we get null anyways, i would use null consistently, here as well

Reply:
oh I can see now why I used undefined, this ref is never passed to react, so we fully control it